orttraining/orttraining/python/experimental/orttrainer_options.py

```diff
@@ -217,6 +226,11 @@ class ORTTrainerOptions(object):
             enables gradient norm clipping for 'AdamOptimizer' and 'LambOptimizer'
         utils.invertible_layer_norm_gradient (bool, default is False):
             enables use of invertible layer norm gradients
+        utils.max_num_pre_training_graph_transformation_steps (int, default is 1):
```

Comment:
Do you feel `max_num_pre_training_graph_transformation_steps` and `enable_gelu_approximation` really belong to `utils` or another category would be best?

Reply:
how about a separate one for ORT graph transformer configuration?